ipaserver/install/server/install.py

```diff
@@ -618,6 +618,14 @@ def install_check(installer):
         # check addresses here, dns module is doing own check
         no_matching_interface_for_ip_address_warning(ip_addresses)
 
+    instance_name = "-".join(options.realm_name.split("."))
```

Comment:
If realm name is not explicitly given as an argument to `--ipa-server-install`, this causes
installation to fail with:
```
The ipa-server-install command failed, exception: AttributeError: 'NoneType' object has no attribute 'split'
```

It should just be `realm_name.split` instead of `options.realm_name.split`, I think.

Reply:
I made this change and tested;  once you make this little fix, it is an ACK from me.

Reply:
ack, thanks.